third_party/pcsc-lite/naclport/common/src/constants.js

```diff
@@ -52,4 +52,18 @@ Constants.REQUESTER_TITLE = 'pcsc_lite_function_call';
 // The constant from the PC/SC-Lite API docs.
 /** @const */
 Constants.PNP_NOTIFICATION = String.raw`\\?PnP?\Notification`;
+
+// Corresponds to `IOCTL_SMARTCARD_VENDOR_IFD_EXCHANGE` in the CCID
```

Comment:
I think these CCID-specific constants should go into a different file, because CCID!=PC/SC (CCID is just one of many drivers that's supported by PC/SC-Lite).

So either create a new file third_party/ccid/.../constants.js. Or maybe it's OK to create a new file `testing-constants.js` next to the current one.

Reply:
> Or maybe it's OK to create a new file testing-constants.js next to the current one.
you mean in smart_card_connector_app/src?

Reply:
that'd work too